2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:00 WARNING  urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7298309690>, 'Connection to tracker.tfile.co timed out. (connect timeout=30)')': /scrape?info_hash=%D0%08%9D%D0%CD%CA%02y%1E%20%FF%B1%ABX%A5t%1AlF%B2
2018-10-12 19:00 WARNING  urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f72d7fbd650>, 'Connection to retracker.spb.ru timed out. (connect timeout=30)')': /scrape?info_hash=%D0%08%9D%D0%CD%CA%02y%1E%20%FF%B1%ABX%A5t%1AlF%B2
2018-10-12 19:01 INFO     remember_rej  download-series-rss Remembering rejection of `A Discovery Of Witches S01E05 INTERNAL 1080p HDTV x264-FaiLED` for 15 minutes
2018-10-12 19:01 INFO     task          download-series-rss Plugin torrent_alive has requested task to be ran again after execution has completed. Reason: Not enough seeds
2018-10-12 19:01 INFO     task          download-series-rss Rerunning the task in case better resolution can be achieved.
2018-10-12 19:15 INFO     exec          move-series     Stdout: find: ‘/mnt/BISMARK/downloads/series/*’: No such file or directory
2018-10-12 19:15 INFO     exec          move-movies     Stdout: find: ‘/mnt/BISMARK/downloads/filmes/*’: No such file or directory
2018-10-12 19:15 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/Supernatural/Supernatural - S14E01 - Stranger in a Strange Land - [720p hdtv h264].mkv was Supernatural
2018-10-12 19:15 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/How to Get Away with Murder/How to Get Away with Murder - S05E03 - The Baby Was Never Dead - [720p hdtv h264].mkv was How to Get Away with Murder
2018-10-12 19:15 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/This Is Us/This Is Us - S03E03 - Katie Girls - [720p webdl h264].mkv was This Is
2018-10-12 19:15 INFO     subtitles     download-subtitles Name computed for /mnt/MOGAMI/Filmes/The Darkest Minds (2018) - [1080p webrip h264].mp4 was The Darkest Minds
2018-10-12 19:15 CRITICAL task          download-subtitles BUG: Unhandled error in plugin subliminal: 'list' object has no attribute 'lower'
2018-10-12 19:15 CRITICAL manager       download-subtitles An unexpected crash has occurred. Writing crash report to /home/braga/flexget/crash_report.2018.10.12.191511348146.log. Please verify you are running the latest version of flexget by using "flexget -V" from CLI or by using version_checker plugin at http://flexget.com/wiki/Plugins/version_checker. You are currently using version 2.15.1
2018-10-12 19:15 WARNING  task          download-subtitles Aborting task (plugin: subliminal)
2018-10-12 19:30 INFO     download      download-series-rss Downloading: A Discovery Of Witches S01E05 INTERNAL 1080p HDTV x264-FaiLED
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno 111] Connection refused
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno 111] Connection refused
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno 111] Connection refused
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno 111] Connection refused
2018-10-12 19:30 WARNING  urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f72982d62d0>: Failed to establish a new connection: [Errno 111] Connection refused',)': /scrape?info_hash=%D0%08%9D%D0%CD%CA%02y%1E%20%FF%B1%ABX%A5t%1AlF%B2
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno -2] Name or service not known
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: [Errno -2] Name or service not known
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  torrent_alive                 Socket Error: timed out
2018-10-12 19:30 WARNING  urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f72987fb0d0>, 'Connection to tracker.tfile.co timed out. (connect timeout=30)')': /scrape?info_hash=%D0%08%9D%D0%CD%CA%02y%1E%20%FF%B1%ABX%A5t%1AlF%B2
2018-10-12 19:30 WARNING  urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f72982d6290>, 'Connection to retracker.spb.ru timed out. (connect timeout=30)')': /scrape?info_hash=%D0%08%9D%D0%CD%CA%02y%1E%20%FF%B1%ABX%A5t%1AlF%B2
2018-10-12 19:31 INFO     transmission  download-series-rss "A Discovery Of Witches S01E05 INTERNAL 1080p HDTV x264-FaiLED" torrent added to transmission
2018-10-12 19:45 INFO     exec          move-movies     Stdout: find: ‘/mnt/BISMARK/downloads/filmes/*’: No such file or directory
2018-10-12 19:45 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/Supernatural/Supernatural - S14E01 - Stranger in a Strange Land - [720p hdtv h264].mkv was Supernatural
2018-10-12 19:45 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/How to Get Away with Murder/How to Get Away with Murder - S05E03 - The Baby Was Never Dead - [720p hdtv h264].mkv was How to Get Away with Murder
2018-10-12 19:45 INFO     subtitles     download-subtitles Name computed for /mnt/YAMATO/Series/This Is Us/This Is Us - S03E03 - Katie Girls - [720p webdl h264].mkv was This Is
2018-10-12 19:45 INFO     subtitles     download-subtitles Name computed for /mnt/MOGAMI/Filmes/The Darkest Minds (2018) - [1080p webrip h264].mp4 was The Darkest Minds
2018-10-12 19:45 CRITICAL task          download-subtitles BUG: Unhandled error in plugin subliminal: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/home/braga/flexget/lib/python2.7/site-packages/flexget/task.py", line 486, in __run_plugin
    return method(*args, **kwargs)
  File "/home/braga/flexget/lib/python2.7/site-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/home/braga/flexget/lib/python2.7/site-packages/flexget/plugins/output/subtitles_subliminal.py", line 197, in on_task_output
    hearing_impaired=hearing_impaired)
  File "/home/braga/flexget/lib/python2.7/site-packages/subliminal/core.py", line 211, in download_best_subtitles
    for s in subtitles], key=operator.itemgetter(1), reverse=True)
  File "/home/braga/flexget/lib/python2.7/site-packages/subliminal/score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/braga/flexget/lib/python2.7/site-packages/subliminal/providers/legendastv.py", line 150, in get_matches
    matches |= guess_matches(video, guessit(self.name, {'type': self.type}))
  File "/home/braga/flexget/lib/python2.7/site-packages/subliminal/subtitle.py", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
